Automattic/jetpack
# Update Get Apps card to reflect the Jetpack app

#### Changes proposed in this Pull Request:

The following changes have been made to the "get apps" section found via _Jetpack_ → _Dashboard_:

* The section is no longer dismissable.
* A QR code is available for users to scan when viewed on desktop, with buttons on mobile.
* Some updates have been made to the design, including tweaks to padding, the new app icon, etc.

#### Other information:


#### Jetpack product discussion

pcdRpT-F0-p2 

#### Does this pull request change what data or activity we track or use?

Yes, downloads from the app buttons may decrease, due to the fact that they'll only available from mobile devices following this change.

#### Testing instructions:

* With this branch checked out, navigate to _Jetpack_ → _Dashboard_.
* Scroll down to the mobile app card and verify that the design has been updated.

#### Screenshots:

<details>
 <summary>Design comparison (web) ⤵️</summary>

Note, the design in Figma used the `SF Pro` font. I didn't specify that in the design to keep consistency with the default font used in other sections on the page, but am happy to change if required. I aimed to match all other sizes/dimensions provided in the Figma.

**Design:**

<img width="879" alt="Screenshot 2022-09-20 at 16 49 20" src="https://user-images.githubusercontent.com/2998162/191305112-0239036b-e743-49ee-adcf-1801713aca0a.png">

**Actual:**

<img width="1023" alt="Screenshot 2022-09-20 at 17 07 18" src="https://user-images.githubusercontent.com/2998162/192515088-34954ff5-4744-4d8d-b530-4061569c33ff.png">

</details>

<details>
 <summary>Before/after (web) ⤵️</summary>

These before/after screenshots are taken in Google Chrome `(105.0.5195.125)` on macOS `(12.6)`.

**Before:**

<img width="1011" alt="Screenshot 2022-09-19 at 13 40 50" src="https://user-images.githubusercontent.com/2998162/191019602-3fa00c62-72db-4436-9936-083c380bdca8.png">

**After:**

<img width="1023" alt="Screenshot 2022-09-20 at 17 07 18" src="https://user-images.githubusercontent.com/2998162/192515088-34954ff5-4744-4d8d-b530-4061569c33ff.png">

</details>

<details>

 <summary>Mobile web ⤵️</summary>

It's worth noting that these screenshots were taken in emulators, rather than from actual devices.

| Google Pixel 5  | iPhone XR |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/192515528-eb6e87a9-9514-4995-af6a-da73e9ac3bbd.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/192515627-5ca3415b-3ec9-4490-a740-e2b0c1a67bbd.png" width="100%"> |
</details>

<details>
 <summary>x-browser testing ⤵️</summary>

The main browser I used for development was Chrome, for which screenshots have already been provided above. In addition, I tested this change across the latest versions of Opera, Firefox, Edge, and Safari on macOS `(12.6)`.

| Opera  | Firefox |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/192515901-f02009ec-15b3-4196-a90c-6135b98cd006.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/192516343-3afe7bf1-2691-40d6-99cf-e262ea1162e4.png" width="100%"> |

| Safari  | Edge |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/192516470-ea3ebcc9-cc9a-496f-a83e-9077772c57d3.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/192516690-08471929-e6bd-4962-ac66-9b99a37ff36e.png" width="100%"> |

</details>
